docs/framework/data/adonet/sql/linq/index.md

```diff
@@ -29,7 +29,7 @@ ms.assetid: 73d13345-eece-471a-af40-4cc7a2f11655
  Provides links to Visual Basic and C# samples.  
   
 ## Related Sections  
- [Language-Integrated Query (LINQ) - C#](../../../../../csharp/programming-guide/concepts/linq/index.md)\
+ [Language-Integrated Query (LINQ) - C#](../../../../../csharp/programming-guide/concepts/linq/index.md)
```

Comment:
that backslash was necessary to break the line
![image](https://user-images.githubusercontent.com/12971179/52390389-e5a22280-2a4c-11e9-9ebc-eb97b85ad8ff.png)